modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_general.scss

```diff
@@ -224,7 +224,8 @@ td.lfr-name-column {
 .portlet-forms {
 	.form-group
 		.alloy-editor-container.input-group
-		.alloy-editor-placeholder.ae-placeholder {
+		.ae-placeholder
```

Comment:
I think We don't have alloyeditor in Forms. I talked with @miltonmc here at the office and he never saw this editor on forms since last year.

Also, this [thread](https://liferay.slack.com/archives/CL8R04Z7X/p1631645043100700) can bring more information about this code.

I think it's safer to just remove these alloyeditor related styles in *-form-web module.